web/cobrands/fixmystreet/assets.js

```diff
@@ -546,6 +538,18 @@ fixmystreet.assets = {
             select_feature_control = new OpenLayers.Control.SelectFeature( asset_layer );
             asset_layer.events.register( 'featureselected', asset_layer, asset_selected);
             asset_layer.events.register( 'featureunselected', asset_layer, asset_unselected);
+            if (options.disable_pin_snapping) {
+                // The pin is snapped to the centre of a feature by the select
+                // handler. We can stop this handler from running, and the pin
+                // being snapped, by returning false from a beforefeatureselected
+                // event handler. This handler does need to make sure the
+                // attributes of the clicked feature are applied to the extra
+                // details form fields first though.
```

Comment:
Do we want to trigger an `assets:selected` event, in case there is a roads layer that might be interacting here as well? Or just note this in case that happens in future.